man/systemd.link.xml

```diff
@@ -144,7 +144,7 @@
       <varlistentry id='property'>
         <term><varname>Property=</varname></term>
         <listitem>
-          <para>A whitespace-separated list of udev property name with its value after a equal
+          <para>A whitespace-separated list of udev property name with its value after an equal
           (<literal>=</literal>). If multiple properties are specified, the test results are ANDed.
```

Comment:
Preexisting, but "equal" → "equals sign".

Reply:
shouldn't that be plural? "... list of property names with their ...."